sdcm/mgmt.py

```diff
@@ -553,22 +558,25 @@ def get_hosts_health(self):
                 list_titles_row = hosts_table[0]
                 host_col_idx = list_titles_row.index("Host")
                 cql_status_col_idx = list_titles_row.index("CQL")
-                ssl_col_idx = list_titles_row.index("SSL")
                 rest_col_idx = list_titles_row.index("REST")
 
                 for line in hosts_table[1:]:
                     host = line[host_col_idx]
                     list_cql = line[cql_status_col_idx].split()
                     status = list_cql[0]
-                    rtt = list_cql[1].strip("()") if len(list_cql) == 2 else "N/A"
+                    cql_rrt_list = findall(r"\(([^)]+)", list_cql[-1])
+                    rtt = cql_rrt_list[0] if len(cql_rrt_list) == 1 else "N/A"
```

Comment:
what ```rtt``` stands for?

Reply:
basically means ping time
https://en.wikipedia.org/wiki/Round-trip_delay_time

Reply:
ping time (or Round Trip Time) should be then described by RTT, not RRT.
please, use meaningful names.